js/serializers/questions.js

```diff
@@ -1,5 +1,5 @@
 define([
-    'coreJS/adapt',
+    'core/js/adapt',
     './scormSuspendDataSerializer'
```

Comment:
Realise this was present before your amends but is `'./scormSuspendDataSerializer'` actually used? It hasn't got an associated variable.